serializer.rst

```diff
@@ -207,52 +126,11 @@ stored in one of the following locations:
 
 .. _serializer-enabling-metadata-cache:
 
-Enabling the Metadata Cache
----------------------------
-
-Metadata used by the Serializer component such as groups can be cached to
-enhance application performance. Any service implementing the ``Doctrine\Common\Cache\Cache``
-interface can be used.
-
-A service leveraging `APCu`_ (and APC for PHP < 5.5) is built-in.
-
-.. configuration-block::
-
-    .. code-block:: yaml
+Configruing the Metadata Cache
+------------------------------
 
-        # config/packages/prod/framework.yaml
-        framework:
-            # ...
-            serializer:
-                cache: serializer.mapping.cache.apc
-
-    .. code-block:: xml
-
-        <!-- config/packages/prod/framework.xml -->
-        <?xml version="1.0" encoding="UTF-8"?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:framework="http://symfony.com/schema/dic/symfony"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd
-                http://symfony.com/schema/dic/symfony
-                http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
-
-            <framework:config>
-                <!-- ... -->
-                <framework:serializer cache="serializer.mapping.cache.apc" />
-            </framework:config>
-        </container>
-
-    .. code-block:: php
-
-        // config/packages/prod/framework.php
-        $container->loadFromExtension('framework', array(
-            // ...
-            'serializer' => array(
-                'cache' => 'serializer.mapping.cache.apc',
-            ),
-        ));
+The metadata for the serializer is automatically cached. To configure the cache,
+configure the ``framework.cache.pools`` key in ``config/packages/framework.yaml``.
```

Comment:
Can't we link to another document here?

Reply:
Ha! I'm happy you said that. No, there is no better link yet, because we're missing some cache docs! When I hit this, I already created #8711 and linked back to this, so we can improve it later :)